content/en/flux/security/slsa-assessment.md

```diff
@@ -0,0 +1,188 @@
+
+- The Flux project uses Git for source code management and the Flux project's repositories are hosted on GitHub under
+  the FluxCD organization.
+- All the Flux maintainers are required to have two-factor authentication enabled and to sign-off all their
+  contributions.
+- The Flux project uses GitHub Actions and GitHub Runners for building all its release artifacts.
+- The build and release process runs in isolation on an ephemeral environment provided by GitHub-hosted runners.
+
+#### Follow a consistent build process
+
+> The producer MUST build their artifact in a consistent manner such that verifiers can form expectations about the
+> build process.
+
+- The build and release process is defined in code (GitHub Workflows and Makefiles) and is kept under version control.
+- The GitHub Workflows make use of GitHub Actions pinned to their Git commit SHA and are kept up-to-date using GitHub
+  Dependabot.
+- All changes to build and release process are done via Pull Requests that must be approved by at least one Flux
+  maintainer.
+- The release process can only be kicked off by a Flux maintainer by pushing a Git tag in the semver format.
+
+#### Distribute provenance
+
+> The producer MUST distribute provenance to artifact consumers.
+
+- The Flux project uses the
+  official [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator) for provenance
+  generation and distribution.
+- The provenance for the release artifacts published to GitHub releases (binaries, SBOMs, deploy manifests, source code)
+  is generated using the `generator_generic_slsa3` GitHub Workflow provided by
+  the [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator).
+- The provenance for the release artifacts published to GitHub Container Registry and to DockerHub (Flux controllers
+  multi-arch container images) is generated using the `generator_container_slsa3` GitHub Workflow provided by
+  the [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator).
+
+### Build Platform Requirements
+
+#### Provenance generation
+
+| Requirement               | Required at SLSA L3 | Met by Flux |
+|---------------------------|---------------------|-------------|
+| Provenance Exists         | Yes                 | Yes         |
+| Provenance is Authentic   | Yes                 | Yes         |
+| Provenance is Unforgeable | Yes                 | Yes         |
+
+> The build process MUST generate provenance that unambiguously identifies the output package by cryptographic digest
+> and describes how that package was produced.
+
+- The Flux project release workflows make use of the
+  official [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator) for provenance
+  generation.
+- The provenance file stores the SHA-256 hashes of the release artifacts (binaries, SBOMs, deploy manifests, source
```

Comment:
```suggestion
- The provenance file stores the SHA-256 hash of the release artifacts (binaries, SBOMs, deploy manifests, source
```

Reply:
It's hashes, because we have multiple assets!

Reply:
See https://github.com/fluxcd/source-controller/blob/main/.github/workflows/release.yml#L22